src/rail/cli/pipe_commands.py

```diff
@@ -131,7 +168,11 @@ def inform(config_file, **kwargs):
     iter_kwargs = project.generate_kwargs_iterable(flavor=flavors, selection=selections)
     ok = 0
     for kw in iter_kwargs:
-        ok |= pipe_scripts.inform_pipeline(project, **kw, **kwargs)
+        ok |= pipe_scripts.run_pipeline_on_single_input(
+            project, "inform",
```

Comment:
should define `pipeline_name = "inform"` and use that variable as the argument here for consistency with the other pipeline calls

Reply:
likewise for subsequent `run_pipeline_on_single_input` calls... or maybe just pass in the name directly in the previous commands? I don't particularly mind either way, but would be good to be consistent